core/domain/exp_services.py

```diff
@@ -1170,6 +1154,9 @@ def update_exploration(
             message starts with the same prefix as the commit message for
             accepted suggestions.
     """
+    if change_list is None:
```

Comment:
is this change made in recent commits?

1. Can you please explain why we need this change?
2. Can you please add relevant param type in the docstring as well?

Reply:
Um I actually don't remember why I made this change but most probably I think None is passed as parameter in update_exploration somewhere in the codebase. Does this change seem wrong or is it okay to add here? If it is okay, I will add the param type in the docstring.

Also this change is not made in the recent commits but actually was made in the very first one

Reply:
I don't have any issue with adding such checks, please do update the docstring.

Reply:
Done